tools/windows-node-installer/test/e2e/common.go

```diff
@@ -0,0 +1,84 @@
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/masterzen/winrm"
+	"github.com/openshift/windows-machine-config-operator/tools/windows-node-installer/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	// winRMPort is the secure WinRM port
+	winRMPort = "5986"
+	// winFirewallCmd will verify if firewall rule is present on the windows node
+	winFirewallCmd = "powershell.exe Get-NetFirewallRule -DisplayName ContainerLogsPort"
```

Comment:
There is no reason for this to be a global variable. It is only needed in `checkForFirewallRule()`